### ISSUE TITLE
[Suggestion] Ability to Limit Quarry Depth

### ISSUE THREAD

**Issue by Neshram** [rwtema/extrautilities_old_issues#459]
Just a thought. It would occasionally be handy if we could limit the depth the quarry digs to, instead of going all the way to to bottom of the world. Perhaps using a fourth Marker, or just stopping at the first bedrock block.

The most immediate example is challenge map mod-packs, where the "mines" may be positioned above other areas not intended for mining, and it would therefore be preferable not to go "through" the enclosing bedrock. Such as in the Material Energy^4 pack, where the vast majority of mine-able materials are positioned within an ME spatial storage room.

**Comment by Tergiver** [comment]
I wish for this also, and Material Energy^4 is exactly the reason why.

It's not fatal there, parcel31u did a good job of A) routing the ME cables and B) providing a nearby alternate access point to those cables (the Chem Lab), so you can re-locate the Spatial IO room. But it would still be nice to have it stop at bedrock.

However you should know that at this time, the Ender Quarry crashes when it attempts to mine any PoorOre, so don't load Miner's Delight.

**Comment by krakentoa** [comment]
Me too!

**Comment by Spongman** [comment]
Agreed. Ender Quarry should definitely not mine above bedrock ceiling or below bedrock floor.
